SU-HSDO/suhumsci
# Hotfix(STN-924): going back to using animationFrame api instead of eventListeners as

## Description
[HOTFIX] On-Page Anchor links not functional

# READY FOR REVIEW

## Summary
Going back to using `animationFrame` API instead of `eventListeners` as it seems to have been disabling the window's ability to scroll.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Add back-to-top button and confirm it works with and without animation enhancements
2. Add Text Area(s) like https://politicalscience.stanford.edu/graduate-program/faq-prospective-phd-students#dept-philosophy
3. Confirm ability to clink page links and scroll to corresponding content.
4. Switch themes and repeat. 
5. Be sure to test with and without animations enabled.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
